src/hotspot/cpu/arm/c1_FrameMap_arm.hpp

```diff
@@ -95,8 +95,6 @@
   }
 
   static int adjust_reg_range(int range) {
-    // Reduce the number of available regs (to free Rheap_base) in case of compressed oops
-    if (UseCompressedOops || UseCompressedClassPointers) return range - 1;
     return range;
   }
```

Comment:
This looks to me like the whole method could be removed?

Reply:
Possibly; but I wanted to keep the change to a minimum. There will be a larger cleanup coming when UseCompressedClassPointers gets removed.

Reply:
On closer inspection, this is still used in arm C1. I would like to keep the changes minimal, so I'll just keep it.